nodejs/awsx/ecs/taskDefinition.ts

```diff
@@ -74,11 +74,11 @@ export abstract class TaskDefinition extends pulumi.ComponentResource {
         this.listeners = {...this.applicationListeners, ...this.networkListeners };
 
         const containerString = containerDefinitions.apply(d => JSON.stringify(d));
-        const family = containerString.apply(s => name + "-" + utils.sha1hash(pulumi.getStack() + containerString));
+        const defaultFamily = containerString.apply(s => name + "-" + utils.sha1hash(pulumi.getStack() + containerString));
 
         this.taskDefinition = new aws.ecs.TaskDefinition(name, {
             ...args,
-            family,
+            family: args.family || defaultFamily,
```

Comment:
this needs to use `utils.ifUndefined`.  that will do the right thing if `Input` is a wrapped `undefined` value.

Reply:
Ok, I will update! thanks